modules/recon/companies-multi/whois_miner_ripe.py

```diff
@@ -0,0 +1,58 @@
+from recon.core.module import BaseModule
+from lxml.etree import fromstring
+from urllib.parse import quote_plus
+import ipaddress
+
+
+class Module(BaseModule):
+
+    meta = {
+        'name': 'Whois Data Miner (ripe.net)',
+        'author': 'Andrey Zhukov from USSC',
+        'version': '1.0',
+        'description': 'Uses the RIPE Whois to harvest companies, locations, netblocks, and contacts associated with the given company search string. Updates the respective tables with the results.',
+        'dependencies': ['lxml'],
```

Comment:
This shouldn't need to be included here as I believe it is included in the framework core. Ideally, the stuff listed here should only be dependencies that the user NEEDS to install prior to use. However, now that I think about it, it might not be a bad idea to standardize including all deps outside the standard library even if they are already included in the core. It would keep things properly separated and prevent having to update a ton of modules if a dep falls out of the core. Thoughts on this @MrJester?

Reply:
Interesting thoughts. Could be good as you said with things falling out of core.